saleor/graphql/account/mutations.py

```diff
@@ -431,35 +431,32 @@ def mutate(cls, root, info, input):
         send_user_password_reset_email(user, site)
 
 
-class AddressCreateInput(AddressInput):
-    user_id = graphene.ID(
-        description='ID of a user to create address for', required=True)
-
-
 class AddressCreate(ModelMutation):
+    user = graphene.Field(
+        User, description='A user instance for which the address was created.')
+
     class Arguments:
-        input = AddressCreateInput(
+        input = AddressInput(
             description='Fields required to create address', required=True)
+        user_id = graphene.ID(
+            description='ID of a user to create address for', required=True)
```

Comment:
Maybe we could also support ```type``` argument which supports creating default address directly just like ```CustomerAddressCreate```? Or we can do in further PRs.